### ISSUE TITLE
Request: Add a showcase in official website if possible

### ISSUE THREAD

**Issue by stefanos82** [robiso/wondercms#263]
First of all, congrats for the amazing job; well done :+1: 

As the title says, can we add a showcase with happy users, if we can?

Most customers I had dealt with in the past were forced to use WordPress, because that's what the local market brainwashed them to go for, but found it very sophisticated for their needs; imagine now with Gutenberg's Full-Site Editing (FSE)...

I would like to show them WonderCMS as a lightweight alternative for their tiny shop, their product catalog (you know, 5 to 10 products the most with contact info), and as their landing page(s).

Is it possible?

**Comment by robiso** [comment]
Hello @stefanos82, we already have something in the lines of what you're describing in our Community:
- https://www.wondercms.com/community/viewforum.php?f=44

If enough users posted their website, we could make a separate SHOWCASE on the official wondercms.com website.
WonderCMS currently doesn't have a shop plugin (we used to have it but it had some vulnerabilities as it was a forked project and we had to remove it as we focused on other features.

I'd love to see someone in the community take off where the development of the shop stopped, that would be amazing!
